oeps/oep-0019-bp-developer-documentation.rst

```diff
@@ -288,7 +288,24 @@ This section summarizes the requirements for long-term developer documentation.
 Format
 ~~~~~~
```

Comment:
FYI: After this merges, I'd like to change the header underlines to match those in the cheatsheet, which also match the Python recommendations.  I know that rST allows for anything that is self-consistent in a doc, but when in the middle of a long doc, I don't like having to scroll and memorize the underlines to try to figure out what we are using for each heading.